types/error-cause/Error.d.ts

```diff
@@ -1,9 +1,9 @@
-import BaseError from './base/Error';
+import BaseError, { ErrorType } from './base/Error';
 
 declare class Error extends BaseError {
-    constructor(reason?: string, options?: { cause?: unknown });
+    constructor(reason?: string, options?: { cause?: ErrorType | undefined });
 
-    cause: unknown;
+    cause: ErrorType | undefined;
```

Comment:
This is wildly incorrect. Any JS value can be thrown, and the builtin error types are ibcirrrct if they restrict this in any way.